Feature spring boot starter template (#10)
harsh-2711/getting-started: backend/Spring Boot/README.md

# Getting Started with Spring Boot Technology

Get started learning Spring boot with templete  and basic funtionalities releted to Spring Boot provided in this folder.

> 

Reviewer: This is not required.

## 📌 Introduction

Spring Boot is an open source Java-based framework used to create a Micro Service. Developing Services with spring Boot is too cool.

Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can “just run”. Most Spring Boot applications need very little Spring configuration.

## 📌 How to get deep dive with Spring Boot?

To start working with Spring Boot, we will require to do basic installations.

[Installing Java](https://www.java.com/en/download/help/download_options.html)<br>
[Installing IntelliJ IDEA](https://www.jetbrains.com/help/idea/installation-guide.html)

Checkout starter project and you will be ready to implement Spring Boot as your backend technology to create Services.<br>

> In this Repository, you will find the cool things about Spring Boot and Spring Boot Starter Projects.Basic REST Services will be made available to this folder .

Reviewer: ```suggestion
In this folder, you will find all the getting-started resources about Spring Boot and Spring Boot Starter Projects. Basic REST Services will be made available to this folder.
```

## 📌 Goal
Provide quick start for projects with flavour of Spring Boot .<br>

> Provide a range of features that are common to large classes of projects

Reviewer: ```suggestion
Provide a range of features that are common to large classes of projects.

```

> ## 📌 Prerequisite to use Spring Boot

Reviewer: Some direct links here to understand Annotations, basics of Spring, etc. will be helpful.

> You should know Java. You should understand usage of Annotations.<br>
> You should understand the basics of Spring framework.<br>
> A basic understanding of JPA.

Reviewer: ```suggestion
- You should know Java. You should understand the usage of Annotations.
- You should understand the basics of Spring framework.<br>
- Should have a basic understanding of JPA.

```

> 

Reviewer: Extra lines not needed.
